File: Android/Tejash11_TejashSeth_2024csit1065_2/Patterns/Pattern6.java
import java.util.*;

public class Main {

  public static void main(String[] args) {
    Scanner scn = new Scanner(System.in);
    int n = scn.nextInt();
    int star = n / 2 + 1, space = 1;
    for (int i = 1; i <= n; i++)
    {
      for (int j = 1; j <= star; j++)
      {
        System.out.print("*\t");
      }
      for (int j = 1; j <= space; j++)
      {
        System.out.print("\t");
      }
      for (int j = 1; j <= star; j++)
      {
        System.out.print("*\t"); 
      }

      if (i <= n / 2)
      {
        space += 2;
        star--;
      }
      else
      {
        star++;
        space -= 2;
      }
      System.out.println();
    }
  }
}

File: Web/sneha-jais_snehaJaiswal_2024cs1084_2/Patterns/Pattern6.cpp
#include <iostream>
using namespace std;
int main()
{
    int n;
    cin >> n;

    int star = (n / 2) + 1;
    int space = 1;

    for (int i = 1; i <= n; i++)
    {
        for (int j = 1; j <= star; j++)
        {
            cout << "*\t";
        }

        for (int j = 1; j <= space; j++)
        {
            cout << "\t";
        }

        for (int j = 1; j <= star; j++)
        {
            cout << "*\t";
        }

        if (i <= n / 2)
        {
            star--;
            space += 2;
        }
        else
        {
            star++;
            space -= 2;
        }
        cout << endl;
    }

    return 0;
}